src/index.html

```diff
@@ -11,7 +11,138 @@
+          max="100"
+          min="12"
+          maxlength="3"
+          id="age">
+        </div>
+        <div class="formField">
+          <label for="date">Full date of birth:</label>
+          <input type="date"
+          name="date"
+          placeholder="mm.dd.yyyy"
+          autocomplete="off"
+          id="date">
+        </div>
+
+        <div class="formField">
+          <label for="agreement">I accept the term of the agreement</label>
+          <input type="checkbox"
+          name="agreement"
+          id="agreement"
+          required>
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Registration</legend>
+        <div class="formField">
+          <label for="email">E-mail:</label>
+          <input type="email"
+          placeholder="email@example.com"
+          name="email"
+          id="email">
+        </div>
+        <div class="formField">
+          <label for="password">Password:</label>
+          <input type="password"
+          name="password"
+          id="password"
+          minlength="5"
+          maxlength="20">
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend>An interesting fact about you!</legend>
+        <div class="formField">
+          <label for="fact">Do you love cats?</label>
+          <input type="radio"
+          name="fact"
+          value="yes"
+          id="fact">
+          <label for="fact">Yes</label>
+          <input
+          type="radio"
+          name="fact"
+          value="no"
+          id="radioNo"
+          >
+          <label for="fact">No</label>
+        </div>
+        <div class="formField">
+          <label for="color">What is your favorite color?</label>
+          <input type="color"
+          name="color"
+          id="color">
+        </div>
+        <div class="formField">
+          <label for="bedtime">What time do you go to bed?</label>
+          <input type="time"
+          name="bedtime"
+          id="bedtime">
+        </div>
+        <div class="formField">
+          <label for="cars">What are your favorite brands of cars?</label>
+          <select name="cars"
+          id="cars"
+          multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+        <div class="formField">
+          <label for="rate">How do you rate our work?</label>
+          <input type="range"
+          name="rate"
+          id="rate">
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Additional info:</legend>
+        <div class="formField">
+          <label for="comments">Comments:</label>
+          <textarea name="comments"
+          id="comments"
+          cols="20"
+          rows="2">
+          </textarea>
+        </div>
+        <label class="fieldForm" for="recommend">Would you recommend us?
```

Comment:
Let's put text on a new line, for better code style
```suggestion
        <label class="fieldForm" for="recommend">
        	Would you recommend us?
```